dags/sources/bigquery.py

```diff
@@ -84,7 +93,10 @@ def schema() -> Optional[ProtocolSchema]:
                 validation="^[a-zA-Z0-9_]{1,1024}$")),
             ("table", str, Field(
                 description="The name of your BigQuery table.",)),
-            
+            ("unique_id", Optional[str], Field(
+                description=f"Unique id column name to be used by BigQuery. Defaults to '{_UNIQUE_ID_DEFAULT_NAME}' when nothing is provided.",
+                default=_UNIQUE_ID_DEFAULT_NAME
```

Comment:
Nit: on the credentials the default value is first and on this one it's second.

Reply:
Nice catch